contrib/node/src/python/pants/contrib/node/tasks/node_repl.py

```diff
@@ -70,3 +70,9 @@ def launch_repl(self, targets):
 
         repl_session = node_repl.run()
         repl_session.wait()
+    # TODO(qsong): Consider not specifying dependency in BUILD
```

Comment:
Not clear what this means. We definitely want to support specifying source-level deps in BUILD files at some point.